docs/how-to-guides/adding-or-reusing-tests.md

```diff
@@ -193,11 +325,11 @@ def your_test(environment: Environment):
     # ... run test
 ```
 
-### Running your own redis cluster for testing
+### Running your own Redis cluster for testing
 
-* Install redis on your computer. If you are a mac user, you should be able to `brew install redis`.
-    * Running `redis-server --help` and `redis-cli --help` should show corresponding help menus.
-* Run `cd scripts/create-cluster` and run `./create-cluster start` then `./create-cluster create` to start the server. You should see output that looks like this:
+* Install Redis on your computer. If you are a mac user, you should be able to `brew install Redis`.
```

Comment:
I think you accidentally swapped all `redis` to `Redis` - this should be `brew install redis`, and below should be e.g. `redis-server`

Reply:
Oh shoot yeah I didn't realize nice catch